issueWork/UI/UX__issue/README.md

```diff
@@ -0,0 +1,27 @@
+# UI / UX issue  
+
+## Changes made:  
+
+> 1. Complete overhall of webpage layout based on previously received feedback. Essentially creating a clean and somewhat minimal style.
+ - ### Laws:  
+   - **Aesthetic-Usability**
+   - **Jakob's Law**
+   - **Hick's Law**
```

Comment:
What decision have you optimized with this new design. How did you optimize it?

Reply:
My main point here with Hick's Law was but removing any unneeded fluff from the portfolio. Giving the hiring manager what they want to see as streamlined as possible. By not hiding the project section behind a lengthy road to get there. They will have more time with my projects which are the real "prof in the pudding" items. And if they chose they can read more about me if I meet their standards to do so through my projects.

Reply:
I don't know what the UX law is. But if you show someone a button, they're gonna click it. I bet you 8 times out of 10 folks are gonna click the learn more button thinking that's where you're telling them to go. I doubt a user is gonna see that scroll, to projects then scroll back to the about. I mean maybe, but you're making a bet that your button isn't enticing, which there may be a better way to showcase your projects if that's what you want to do.